src/modules/Shared/components/SideBar/SideBar.tsx

```diff
@@ -55,23 +55,23 @@ export default function SideBar() {
           <MenuItem
             active={window.location.pathname === "/usersList"}
             icon={<i className="fa-regular fa-address-card"></i>}
-            component={<NavLink to="/usersList" />}
+            component={<NavLink to="usersList" />}
```

Comment:
I don't reccommend having `List` at the end as it can be inffered by the plural name.